src/passport-pinejs/passport-pinejs.ts

```diff
@@ -54,7 +54,7 @@ const setup: ConfigLoader.SetupFunction = async (app: Express.Application) => {
 		passport.use(new LocalStrategy(checkPassword));
 
 		login = (fn) => (req, res, next) =>
-			passport.authenticate('local', (err, user?) => {
+			passport.authenticate('local', (err: any, user?: any) => {
```

Comment:
Why this change?

Reply:
TSC complains.

Reply:
I'll remove it from this PR and make another explicitly for this TSC complains